search-books/src/main/java/nix/education/request/BookRequest.java

```diff
@@ -0,0 +1,22 @@
+package nix.education.request;
+
+import java.util.Date;
+import java.util.List;
+import lombok.Data;
+import lombok.SneakyThrows;
+import nix.education.util.DateFormat;
+
+@Data
+public class BookRequest {
```

Comment:
I don't see a big necessity in this class. We need to perform a manual mapping because of this in SavingBookService. Can we use the Book class instead?

Reply:
Yes, maybe we can, I suppose we'll be have some problems when we want to add new book to data base.

Reply:
This class deleted. Post request use the Book class